.github/workflows/regtest.yml

```diff
@@ -53,9 +61,40 @@ jobs:
               :polaris-server:quarkusAppPartsBuild --rerun \
               -Dquarkus.container-image.build=true
 
+      - name: Save partial Gradle build cache
```

Comment:
We should only store the Gradle caches from `gradle.yaml`, not from other workflows.

It's doesn't make much sense either, because the cache contents are only read, but never modified.